Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/asyncio/base_events.py", line 646, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/hostedtoolcache/Python/3.10.6/x64/lib/python3.10/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import matplotlib.pyplot as plt
# income for each quarter
net_income = [40, 30, 30, 35]
plt.plot(net_income)

profit = [0.5 * income in net_income]
plt.plot(profit)

# with current profits I would expect
dividends = 20 + 15 + 15 + 17.5
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNameError[0m                                 Traceback (most recent call last)
Input [0;32mIn [4][0m, in [0;36m<cell line: 6>[0;34m()[0m
[1;32m      3[0m net_income [38;5;241m=[39m [[38;5;241m40[39m, [38;5;241m30[39m, [38;5;241m30[39m, [38;5;241m35[39m]
[1;32m      4[0m plt[38;5;241m.[39mplot(net_income)
[0;32m----> 6[0m profit [38;5;241m=[39m [[38;5;241m0.5[39m [38;5;241m*[39m [43mincome[49m [38;5;129;01min[39;00m net_income]
[1;32m      7[0m plt[38;5;241m.[39mplot(profit)
[1;32m      9[0m [38;5;66;03m# with current profits I would expect[39;00m

[0;31mNameError[0m: name 'income' is not defined
NameError: name 'income' is not defined

